<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2" rowspan="4">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="ScaleChange">
        <property name="text">
         <string>Scale Change</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ScaleChangeBack">
        <property name="text">
         <string>Scale Change Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Rotate">
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RotateBack">
        <property name="text">
         <string>Rotate Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="Rectangle">
      <property name="text">
       <string>Rectangle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="toLeft">
      <property name="text">
       <string>To Right</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="Perimetr">
        <property name="text">
         <string>Perimetr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Area">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CenterOfMass">
        <property name="text">
         <string>Center Of Mass</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="toRight">
      <property name="text">
       <string>To Left</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="4">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Периметр:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Площадь:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Координаты:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Square">
      <property name="text">
       <string>Square</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Rhomb">
      <property name="text">
       <string>Rhomb</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Round">
      <property name="text">
       <string>Round</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Triangle">
      <property name="text">
       <string>Triangle</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Hexagon">
      <property name="text">
       <string>Hexagon</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="FivePointedStar">
      <property name="text">
       <string>5 Star</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="SixPointedStar">
      <property name="text">
       <string>6 Star</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="EightPointedStar">
      <property name="text">
       <string>8 Star</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
